internal/cli/kraft/lib/create/create.go

```diff
@@ -0,0 +1,315 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/erikgeiser/promptkit/textinput"
+	"github.com/spf13/cobra"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/packmanager"
+	"kraftkit.sh/unikraft/lib/template"
+)
+
+type CreateOptions struct {
+	ProjectName     string `long:"project-name" usage:"Set the project name to the template"`
+	LibraryName     string `long:"library-name" usage:"Set the library name to the template"`
+	LibraryKName    string `long:"library-kname" usage:"Set the library kname to the template"`
```

Comment:
What is `LibraryKName`? 😅

Reply:
Figured it out in the meantime

Maybe we should rename to `LibraryKConfigName`

Reply:
@craciunoiuc I just tried to replicate the attributes same as `pykraft` look [here](https://github.com/unikraft/pykraft/blob/c648d2ea1d360685502b94727bd113a13b0f82ee/kraft/lib/lib.py#L305)